### PR TITLE
Update extendOrdWithCustom.js

### PR DESCRIPTION
**Pull request by RamIndia**
Minor condition check

### CODE REVIEW

File: lib/extendOrdWithCustom.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ function compareAndHandleCustomORDContentWithExistingContent(ordContent, customO
 }
 
 function getCustomORDContent(appConfig) {
-    if (appConfig.env.customOrdContentFile) {
+    if (appConfig.env?.customOrdContentFile) {
```
Comment (zongqichen) [inline lib/extendOrdWithCustom.js:50]: Hi @RamIndia, why do you need this question mark? You can see the [code](https://github.com/cap-js/ord/blob/main/lib/ord.js#L74), there is always env in appConfig.

**Review by aramovic79 — COMMENTED** [review]
I would suggest first to review/approve the PR 71 and then to include this change.

File: lib/extendOrdWithCustom.js (hunk 1 above)
Comment (Fannon) [reply lib/extendOrdWithCustom.js:50]: Is this really given? What happens if there's no `ord` dictionary in the `.cdsrc` file? But if you want to fall back to a non-null value, you could do the "default" fallback in the linked file like:

```js
env: cds.env["ord"] || {},
```

Comment (zongqichen) [reply lib/extendOrdWithCustom.js:50]: Good point, I create an issue: https://github.com/cap-js/ord/issues/74

**Review by zongqichen — APPROVED** [review]
I test the case without ord in .cdsrc.json, this fix is necessary to prevent error raised.

**Pull request merged**